autorest/retriablerequest_1.x.go

```diff
@@ -0,0 +1,39 @@
+// +build !go1.8
```

Comment:
I did too as the docs here are pretty bad.  This translates to "versions less than 1.8".